src/nodes/mod.rs

```diff
@@ -20,6 +23,12 @@ pub(super) fn plugin(app: &mut App) {
 #[derive(PoolLabel, PartialEq, Eq, Debug, Hash, Clone, Default)]
 pub struct SteamAudioPool;
 
+#[derive(PoolLabel, PartialEq, Eq, Debug, Hash, Clone, Default)]
+pub struct SteamAudioReverbPool;
```

Comment:
I wonder if this should really be a separate pool. We might consider adding the send node before the `SteamAudioNode` in the `SteamAudioPool`.